### PR TITLE
GeoJSON OSS-Fuzz fuzzer

### PR DESCRIPTION
**Pull request by Komzpa**
Includes fix for leaks found in first 20 minutes.

### CODE REVIEW

**Comment by pramsey** [comment]
Seems OK reading through. I like the idea of fuzzers but have found tracking down the causes of failures immensely frustrating and hard. If you have a recipe running from "google ticket" to "here are the actual inputs that caused the issue" I would love to see it written down.

**Comment by Komzpa** [comment]
to run fuzzer do something like this
```
git clone https://github.com/google/oss-fuzz
cd oss-fuzz
python infra/helper.py build_image postgis
python infra/helper.py build_fuzzers --sanitizer address postgis /home/kom/proj/postgis/ 
python infra/helper.py run_fuzzer postgis geojson_import_fuzzer
... wait 2 hours ...
==17==ERROR: LeakSanitizer: detected memory leaks
...stacktrace...
artifact_prefix='./'; Test unit written to ./leak-7c9eeff6edcee3fc53f8be8e1981b7538ea90729

cat build/out/postgis/leak-7c9eeff6edcee3fc53f8be8e1981b7538ea90729
*feed into input function as with usual debug process*
```

File: fuzzers/geojson_import_fuzzer.cpp [hunk 1]
```diff
@@ -0,0 +1,75 @@
+/******************************************************************************
+ *
+ * Project:  PostGIS
+ * Purpose:  Fuzzer
+ * Author:   Even Rouault, even.rouault at spatialys.com
```
Comment (Algunenano) [inline fuzzers/geojson_import_fuzzer.cpp]: Late to the party, but I guess this should be 2019 + your name, or the generic Postgis header right?

Comment (Algunenano) [reply fuzzers/geojson_import_fuzzer.cpp]: Same for the other files

**Pull request closed without merging**